src/sentry/api/validators/external_actor.py

```diff
@@ -7,6 +8,10 @@
 EXTERNAL_ID_LENGTH_MIN = 1
 EXTERNAL_ID_LENGTH_MAX = 64
 
+EXTERNAL_NAME_LENGTH_MIN = 1
+EXTERNAL_NAME_LENGTH_MAX = 255
```

Comment:
This was set somewhat arbitrarily, but needs to be long since teams can be nested.